lib/error_tracker.rb

```diff
@@ -1,12 +1,13 @@
 module ErrorTracker
 
   # capture and report an exception to Sentry via Raven, setting user context & params as needed
-  def self.report_exception(exception, user, extra_context={})
+  def self.report_exception_with_context(exception, user, *context_objects)
```

Comment:
If we're not preserving the old method, I would just keep the name the same.  Basically saying 'this is the standard way our application reports exceptions'

Reply:
Fair point.

Reply:
4c4f415